src/CampaignAdmin.php

```diff
@@ -287,14 +288,24 @@ protected function getChangeSetItemResource(ChangeSetItem $changeSetItem)
             $referencedBy = [];
             foreach ($referencedItems as $referencedItem) {
                 $referencedBy[] = [
-                    'href' => $this->SetLink($referencedItem->ID)
+                    'href' => $this->SetLink($referencedItem->ID),
+                    'ObjectID' => $referencedItem->ObjectID
                 ];
             }
             if ($referencedBy) {
-                $hal['_links']['referenced_by'] = $referencedBy;
+                $hal['links']['referenced_by'] = $referencedBy;
             }
         }
 
+        $referToItems = $changeSetItem->findReferenced();
+        $referTo = [];
+        foreach ($referToItems as $referToItem) {
+            $referTo[] = [
+                'ID' => $referToItem->ID
```

Comment:
You cannot identify objects only by ID; Different classes can have the same ID. E.g. SiteTree.ID = 1 and File.ID = 1